compiler/src/dotty/tools/repl/Rendering.scala

```diff
@@ -27,32 +27,9 @@ private[repl] class Rendering(parentClassLoader: Option[ClassLoader] = None):
   var myClassLoader: AbstractFileClassLoader = uninitialized
 
   private def pprintRender(value: Any, width: Int, height: Int, initialOffset: Int)(using Context): String = {
-    def fallback() =
```

Comment:
@hamzaremmal why was this reflective call dropped? without it pprint just falls back to println